cmake/test.cmake

```diff
@@ -1,5 +1,15 @@
 set(MAYA_USD_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 
+if(MayaUsd_FOUND)
+    #Add MAYAUSD_MOD_PATH (the path where maya USD .mod file is) to the MAYA_MODULE_PATH
+    # Get the current value of the environment variable
+    set(CURRENT_MAYA_MODULE_PATH $ENV{MAYA_MODULE_PATH})
+    # Append the new path to the current value
+    set(MAYA_MODULE_PATH "${CURRENT_MAYA_MODULE_PATH}:${MAYAUSD_MOD_PATH}")
```

Comment:
Does Windows accept `:` in this env var? I think it would be `;` for Windows and `:` for Mac/Linux.